development_scripts/ec_tools_development_pyCOox_data.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan 25 22:02:57 2021
+
+@author: scott
+"""
+from pathlib import Path
+from matplotlib import pyplot as plt
+
+from ixdat import Measurement
+
+plt.close("all")
+
+if False:
+    data_dir = Path(
+        "~/Dropbox/ixdat_resources/20B12_Data_Analysis_Workshop/example data set/"
+    )
+
+    ocp_file = data_dir / "01_Trimi1_cont_02_OCV_C01.mpt"
+    cv_file = data_dir / "01_Trimi1_cont_03_CVA_C01.mpt"
+    cp_file = data_dir / "01_Trimi1_cont_04_CP_C01.mpt"
+
+    ocp_meas = Measurement.read(ocp_file, reader="biologic", name="Pt_demo_ocp")
+    print("read ocp file!")
+    cv_meas = Measurement.read(cv_file, reader="biologic", name="Pt_demo_cv")
+    print("read cv file!")
+    cp_meas = Measurement.read(cp_file, reader="biologic", name="Pt_demo_cp")
+    print("read cp file!")
+
+    ocp_id = ocp_meas.save()
+    cv_id = cv_meas.save()
+    cp_id = cp_meas.save()
+else:
+    ocp_meas = Measurement.get(3)
```

Comment:
It may not matter, but I guess these id numbers aren't portable

Reply:
Right. But they fit in human short-term memory :D

Reply:
Oh. I meant more that I though that the fact that the turned out to be number 3, 4 and 5 is an artifact of the time at which you ran it and that the test script might need to store those ids somehow, for someone else to be able to run the script. But as I said, it may be a bit of a stretch for now.